examples/publication-briefing.post.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Erstbestückung Wein-Aktionsfläche Rümlang (596) KW 22/2020",
+    "week": {
+        "year": 2020,
+        "number": 22
+    },
+    "store": "596",
```

Comment:
Type mismatch (`string` vs. `int` in response).

Reply:
both is `int`, I'll change here